publish-docker-images.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash -e
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+export DOCKER_IMAGE_TAG=latest
+
+docker login -u ${DOCKER_USER_ID?} -p ${DOCKER_PASSWORD?}
+
+./gradlew assemble mysqlComposeBuild mysqlComposePush
```

Comment:
Why is mysql being pushed?

Reply:
Chris, I cannot find where did I get it, I will remove it